starter-code/js/index.js

```diff
@@ -3,15 +3,32 @@ function deleteItem(e){
 }
 
 function getPriceByProduct(itemNode){
-
+  var priceUnity = Number(document.querySelectorAll('.unity-price')[i].innerText);
+  return priceUnity;
 }
 
 function updatePriceByProduct(productPrice, index){
+    var productPrice = document.querySelectorAll('.unity-price');
+  for(i = 0; i < productPrice.length; i++){
+    var quantityUnity = Number(document.querySelectorAll('.input-qty')[i].value);
+    var total = quantityUnity * getPriceByProduct();
+    document.querySelectorAll('.total-price')[i].innerText = total
+  }
+}
 
+function foo (array){
+  var acc;
+  for (i = 0; i < document.querySelectorAll('.total-price').length; i++){
+    acc = acc + array[i];
+  }
+  return acc;
 }
 
 function getTotalPrice() {
-
+  var totalPrice = document.querySelectorAll('.total-price');
+    var finalPrice = foo(totalPrice);
+    document.querySelectorAll('.final-total').innerText = finalPrice;
```

Comment:
This returns an array, so you can't modify its innerText. Doesn't matter if you only have one element, you need to access to that first element:
```
document.querySelectorAll('.final-total')[0].innerText = finalPrice;
```
In addition, this will remove the rest of the text: "Total Price: $".